rust_src/src/functions.rs

```diff
@@ -20,27 +19,31 @@ use lisp::LispObject;
 #[no_mangle]
 pub static mut lispsym: EmacsInt = 0;
 
-#[mock]
-extern "C" {
-    pub fn Fcons(car: LispObject, cdr: LispObject) -> LispObject;
-    pub fn Fsignal(error_symbol: LispObject, data: LispObject);
-    pub fn make_string(s: *const c_char, length: ptrdiff_t) -> LispObject;
-    pub fn make_unibyte_string(s: *const c_char, length: ptrdiff_t) -> LispObject;
-}
+//#[mock]
+//extern "C" {
+//    pub fn Fcons(car: LispObject, cdr: LispObject) -> LispObject;
+//    pub fn Fsignal(error_symbol: LispObject, data: LispObject) -> !;
+//    pub fn make_string(s: *const c_char, length: isize) -> LispObject;
+//    pub fn make_unibyte_string(s: *const c_char, length: isize) -> LispObject;
+//}
 
```

Comment:
We should remove this code and the mock derive dependency if we're no longer using it.